core/snmp/api/src/main/java/org/opennms/netmgt/snmp/InetAddrUtils.java

```diff
@@ -31,6 +31,9 @@
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 
+/**
+ * @deprecated Use {@link InetAddressUtils} instead.
+ */
 public abstract class InetAddrUtils {
```

Comment:
It is deprecated because our ```InetAddressUtils``` is a widely-used class for these sorts of operations, hence the comment.